examples/typescript/exec/exec-example.ts

```diff
@@ -11,7 +9,7 @@ exec.exec(
     'default',
     'nginx-4217019353-9gl4s',
     'nginx',
-    command,
+    ['ls, '-al', '.'],
```

Comment:
What would happen if you run it with: `<typescript> ./examples/typescript/exec/exec-example.ts 'ls -al'` - that should work, right?

Reply:
I was getting “bin not found” errors doing as you suggest/ask. I presume the string vs array is encoded differently in the resultant url

Reply:
I see, I think I understand.
I think this is fine as this is only an example. Thank you.